docs/2101UserGuide.md

```diff
@@ -4,35 +4,112 @@ style: guide
    Some example commands you can try:
 
     * `list` : Lists all members and events.
 
-    * `createMember n/CHU WEI RONG g/Male p/98765432 e/chuweirongrocks@gmail.com a/19 Kent Ridge Crescent, Singapore 119278` : Creates a member named `CHU WEI RONG` in CCACommander.
+    * `createMember n/Lim Jun Jie g/Male p/98765432 e/limjunjie@gmail.com a/19 Kent Ridge Crescent, Singapore 119278`: Creates a member named Lim Jun Jie in CCACommander.
 
     * `deleteMember 3` : Deletes the 3rd member shown in the current list.
 
-1. Refer to the [Features](#features) below for details of each command.
+6. You can refer to the [Features](#features) below for the details of each command.
+
+<div markdown="block" class="alert alert-primary">
+
+:bulb: You can use the `clear` command to remove the sample data.
+</div>
+
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Navigating User Interface
+![UI_breakdown](images/ui_breakdown.png)
+Here are the important components of CCACommander that you will use and interact with.
+
+**Menu**: The CCACommander menu is located at the top-left corner of your screen. Click the sub-menu items to execute the following
+1. File: Exit the program
+2. Theme: Toggle between light and dark themes
+
+Light | Dark
+--------|--------
+![LightTheme](images/light_theme.png) | ![DarkTheme](images/dark_theme.png)
+
+3. Help: Access our user guide
+
+**Command Box**: Type in your text commands. <br>
+**Result Display**: View messages from CCACommander regarding your commands. <br>
+**Member List**: View the members stored in CCACommander. <br>
+**Event List**: View the events stored in CCACommander. <br>
+**Storage Location**: Locate where your CCACommander data is stored in your computer.
+
+### Command History
+You can quickly recall previously entered commands for convenience as CCACommander stores almost all (see the info card below to see what commands are stored) commands entered by the user. <br/>
+Upon selecting the command box, you can use the <kbd>↑</kbd> to <kbd>↓</kbd> key to navigate through the commands entered .
+
+Example:
+1. You have just marked attendance for the first member in the member list using the editEnrolment m/1 e/1 r/Present command.
+2. You can then use <kbd>↑</kbd> to get editEnrolment m/1 e/1 r/Present
+3. You can edit the member index to get editEnrolment m/2 e/1 r/Present to mark the second member as present.
+
+<div markdown="block" class="alert alert-info">:information_source: CCACommander will not store 2 of the same commands if they were entered twice in a row. Likewise, "commands" containing exclusively of blank spaces will not be stored as well.
+
+Upon entering the <kbd>⌫ Backspace</kbd> or the <kbd>↵ Enter</kbd> key, users will have to re-navigate from the most recent command as it was treated as an edit or entry of command.
```

Comment:
```suggestion
Upon entering the <kbd>⌫ Backspace</kbd> or the <kbd>↵ Enter</kbd> key, users will have to re-navigate from the most recent command as it will be treated as an edit or entry of command.
```